pkg/pillar/types/volumetypes.go

```diff
@@ -430,3 +430,36 @@ func (status VolumeRefStatus) LogDelete(logBase *base.LogObject) {
 func (status VolumeRefStatus) LogKey() string {
 	return string(base.VolumeRefStatusLogType) + "-" + status.Key()
 }
+
+// VolumeCreatePending is temporary store for volumes that are creating
+// After successful creating operation we should delete this object
+type VolumeCreatePending struct {
```

Comment:
Would it be useful to add the log hooks for this type? That way the publish and unpublish will be logged automatically.

Reply:
added